examples/operator-quickstart/01-Install.ipynb

```diff
@@ -0,0 +1,489 @@
+   "metadata": {},
+   "source": [
+    "## Install the Feast Operator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "namespace/feast-operator-system created\n",
+      "customresourcedefinition.apiextensions.k8s.io/featurestores.feast.dev created\n",
+      "serviceaccount/feast-operator-controller-manager created\n",
+      "role.rbac.authorization.k8s.io/feast-operator-leader-election-role created\n",
+      "clusterrole.rbac.authorization.k8s.io/feast-operator-featurestore-editor-role created\n",
+      "clusterrole.rbac.authorization.k8s.io/feast-operator-featurestore-viewer-role created\n",
+      "clusterrole.rbac.authorization.k8s.io/feast-operator-manager-role created\n",
+      "clusterrole.rbac.authorization.k8s.io/feast-operator-metrics-auth-role created\n",
+      "clusterrole.rbac.authorization.k8s.io/feast-operator-metrics-reader created\n",
+      "rolebinding.rbac.authorization.k8s.io/feast-operator-leader-election-rolebinding created\n",
+      "clusterrolebinding.rbac.authorization.k8s.io/feast-operator-manager-rolebinding created\n",
+      "clusterrolebinding.rbac.authorization.k8s.io/feast-operator-metrics-auth-rolebinding created\n",
+      "service/feast-operator-controller-manager-metrics-service created\n",
+      "deployment.apps/feast-operator-controller-manager created\n",
+      "deployment.apps/feast-operator-controller-manager condition met\n"
+     ]
+    }
+   ],
+   "source": [
+    "## Use this install command from a release branch (e.g. 'v0.43-branch')\n",
+    "!kubectl apply -f ../../infra/feast-operator/dist/install.yaml\n",
+    "\n",
+    "## OR, for the latest code/builds, use one the following commands from the 'master' branch\n",
+    "# !make -C ../../infra/feast-operator install deploy IMG=quay.io/feastdev-ci/feast-operator:develop FS_IMG=quay.io/feastdev-ci/feature-server:develop\n",
+    "# !make -C ../../infra/feast-operator install deploy IMG=quay.io/feastdev-ci/feast-operator:$(git rev-parse HEAD) FS_IMG=quay.io/feastdev-ci/feature-server:$(git rev-parse HEAD)\n",
+    "\n",
+    "!kubectl wait --for=condition=available --timeout=5m deployment/feast-operator-controller-manager -n feast-operator-system"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install the Feast services via FeatureStore CR\n",
+    "Next, we'll use the running Feast Operator to install the feast services. Apply the included [reference deployment](feast.yaml) to install and configure Feast."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "secret/feast-data-stores created\n",
+      "featurestore.feast.dev/example created\n"
+     ]
+    }
+   ],
+   "source": [
+    "!kubectl apply -f feast.yaml"
```

Comment:
WDYT if we can mention the namespace here 
`kubectl apply -f feast.yaml -n feast` just to make sure use using right namespace.

Reply:
instead, i'll add the namespace to the individual yaml objects.

Reply:
done, good call